src/components/HelpMenuItems.tsx

```diff
@@ -102,4 +110,45 @@ const HelpMenuItems = () => {
   );
 };
 
+const tourMap = {
+  [createDataSamplesPageUrl()]: TourTrigger.Connect,
+  [createTestingModelPageUrl()]: TourTrigger.TrainModel,
+  // No UI to retrigger MakeCode tour
```

Comment:
Perhaps, when re-triggering a tour via the button, we should clear the settings relating to subsequent tours.

Reply:
I still like this plan this morning.